resources/user-guides/index.md

```diff
@@ -67,7 +67,7 @@ title: "User Guides"
                 </div>
 <div class="row start-xs text-center" markdown="1">
 
-User Guides for Mining coming soon.
+[How to mine on a pool](mine-to-pool.html)
```

Comment:
Maybe something like 'how to mine on a pool with xmr-stak (cpu or gpu)' ? . Or you can just add a quick reference to stak-gpu in the body